src/components/EmployeePage.jsx

```diff
@@ -59,16 +61,19 @@ const EmployeePage = props => {
                             DefaultEmployeeImage
                         }
                     />
+                    <Fragment>
```

Comment:
React har syntakssukker for `<>` og `</>` i stedet for `<Fragment>` og `</Fragment>` men det er selvfølgelig en smakssak om man ønsker å bruke det.

Akkurat her trenger du vel strengt tatt ikke et Fragment i det hele tatt.

Reply:
Det var noe trøbbel med hvis employee.testimonial var null (undefined) så eksploderte alt. Med Fragment rundt fikk jeg ikke det problemet 🤷‍♂

Reply:
Jøss... aldri skjedd med meg. Jaja, skader jo ikke heller.